### PR TITLE
clean ai bootcamp blog

### PR DESCRIPTION
**Pull request by lijiarui**
- change sequence 
- add bilibili link

### CODE REVIEW

File: docs/_posts/2019-12-20-microsoft-ai-global-training-camp-ends-in-beijing.md [hunk 1]
```diff
@@ -77,7 +81,13 @@ Wish you were here.
 
 ![image-25](/assets/2019/microsoft-ai-global-training-camp/image-25.png)
 
-本次活动由DataFun&PNP&句子互动全程落地支持。
+本次活动由Microsoft MVP & DataFun & Plug and Play & Pre-Angel & 句子互动全程落地支持。
```
Comment (windmemory) [inline docs/_posts/2019-12-20-microsoft-ai-global-training-camp-ends-in-beijing.md:84]: Let's put one space before `Microsoft MVP` and one space after `句子互动`.

File: docs/_posts/2019-12-20-microsoft-ai-global-training-camp-ends-in-beijing.md [hunk 2]
```diff
@@ -77,7 +81,13 @@ Wish you were here.
...
+
+### 微软MVP项目介绍
+
+微软最有价值专家Microsoft Most Valuable Professional（MVP）是微软公司授予第三方技术专业人士的一个全球奖项。27年来，世界各地的技术社区领导者，因在线上和线下的技术社区中分享专业知识和经验而获得此奖项。
```
Comment (windmemory) [inline docs/_posts/2019-12-20-microsoft-ai-global-training-camp-ends-in-beijing.md:88]: Space before `Microsoft`

**Review by windmemory — CHANGES_REQUESTED** [review]
I think it would be better to wrap english words with spaces.

**Review by huan — APPROVED** [review]

**Comment by huan** [comment]
Thanks for the update.

**Pull request merged**